docs/Using-Operations-Service.md

```diff
@@ -502,6 +505,50 @@ PostApprovalScreen types:
 - `REDIRECT` providing text for button, countdown, and redirection URL
 - `GENERIC` may contain any object
 
+Definition of `WMTProximityCheck`:
+
+```swift
+public class WMTProximityCheck: Codable {
+    /// Tha actual Time-based one time password
+    public let totp: String
+    /// Type of the Proximity check
+    public let type: WMTProximityCheckType
+    /// Timestamp when the operation was scanned (QR Code) or delivered to the device (Deeplink)
+    public let timestampRequested: Date
+}
+```
+
+WMTProximityCheckType types:
+
+- `qrCode` TOTP was scanned from QR code
+- `deeplink` TOTP was delivered to the app via Deeplink
+
+
+## TOTP WMTProximityCheck
+
+Two-Factor Authentication (2FA) using Time-Based One-Time Passwords (TOTP) in the Operations Service is facilitated through the use of WMTProximityCheck. This allows secure approval of operations through QR code scanning or deeplink handling.
+
+This TOTP-based WMTProximityCheck enhances the security of the approval process, providing a robust mechanism for 2FA in the Operations Service.
+
+- QR Code Flow:
+
+When the `WMTUserOperation` contains a `WMTPreApprovalScreen.qr`, the app should open the camera to scan the QR code before confirming the operation. Use the camera to scan the QR code containing the necessary data payload for the operation.
+
+- Deeplink Flow:
+
+When the app is launched via a deeplink, preserve the data from the deeplink to extract the relevant information. When operations are loaded compare the operation ID from the deeplink data to the operations within the app to find a match.
+
+- Assign TOTP and Type for the Operation
+Once the QR code is scanned or match from deeplink is found, create a `WMTProximityCheck` with:
+    - `totp`: The actual Time-Based One-Time Password.
+    - `type`: Set to `WMTProximityCheckType.qrCode` or `WMTProximityCheckType.deeplink`.
+    - `timestampRequested`: The timestamp when the QR code was scanned (by default, it is created as the current timestamp).
+
+- Authorizing the WMTProximityCheck
+When authorization, the SDK will by default add `timestampSigned` to the `WMTProximityCheck` object. This timestamp indicates when the operation was signed.
+
+This TOTP-based WMTProximityCheck enhances the security of the approval process, providing a robust mechanism for 2FA in the Operations Service.
```

Comment:
Remove this line, this is a vague business description that adds nothing to the technical description of the feature. Otherwise OK